db/schema.rb

```diff
@@ -203,6 +204,7 @@
     t.datetime "updated_at", null: false
     t.uuid "workflow_execution_id"
     t.uuid "sample_id"
+    t.datetime "attachments_updated_at"
```

Comment:
I don't know a lot about data exports, but does it make any sense to add the new column "attachments_updated_at" to it? That's the only other model I can think of that has attachments.

Reply:
I don't think we need it for the data exports. It's moreso for the tables where we want to show when the files were last updated for a sample

Reply:
It isn't needed for data exports as that uses ActiveStorage::Attachment and not our Attachment model as it is just storing a single zip file which we don't need extended metadata for. The column is only needed for models which have `has_many :attachments, as: attachable...`

Reply:
Does that mean the column shouldn't be on workflow executions?

Reply:
Currently workflow_executions doesn't have that setup but I asked Deep to add it here so that Jeff can utilize it in the workflow completion service.